src/app/components/sidebar/sidebar.controller.js

```diff
@@ -9,39 +9,41 @@ angular.module('transmartBaseUi')
     .controller('SidebarCtrl', ['$scope', 'StudyListService', 'EndpointService',
         function ($scope, StudyListService, EndpointService) {
 
-            $scope.publicStudies = [];
-            $scope.privateStudies = [];
+            var ws = this;
```

Comment:
not sure how @rnugraha feels about the naming conventions such as ws, hc, I'll wait for his comments and merge